vvm/install.py

```diff
@@ -180,7 +189,8 @@ def get_installable_vyper_versions(headers: Dict = None) -> List[Version]:
         asset = next((i for i in release["assets"] if _get_os_name() in i["name"]), False)
         if asset:
             version_list.append(version)
-    return sorted(version_list, reverse=True)
+
+    return (_installable_vyper_versions := sorted(version_list, reverse=True))
```

Comment:
bit hard to parse since `_installable_vyper_versions` is a global
```suggestion
    _installable_vyper_versions = sorted(version_list, reverse=True)
    return _installable_vyper_versions
```